src/ui/CoverageProgress/CoverageProgress.tsx

```diff
@@ -0,0 +1,20 @@
+import Progress from 'ui/Progress'
+import { progressColors, progressVariants } from 'ui/Progress/Progress'
+
+const CoverageProgress = ({
+  amount,
+  color = 'primary',
+  variant = 'default',
+}: {
+  amount: number
```

Comment:
This might break when the hooks return null on a type validation. Maybe we can do amount: number | null | undefined. Then you don't need the  /* @ts-expect-error */ in the spec file either

Reply:
ex: useRepoCommitContents